delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/Elem.scala

```diff
@@ -39,7 +96,15 @@ object Elem {
     * @tparam A
     *   the value type of the element
     */
-  final case class SuccessElem[+A](ctx: ElemCtx, value: A) extends Elem[A]
+  final case class SuccessElem[+A](
```

Comment:
It feels like we could drop `Envelope` at some point (not now) and just use `SuccessElem` instead